src/server/pfs/server/driver.go

```diff
@@ -1413,7 +1398,7 @@ func (d *driver) filePathFromEtcdPath(etcdPath string) string {
 	split := strings.Split(trimmed, "/")
 	// we only want /path/to/file so we use index 3 (note that there's an "" at
 	// the beginning of the slice because of the lead /)
-	return path.Join(split[3:]...)
+	return path.Join(split[6:]...)
```

Comment:
I'm assuming this changed from `3` to `6` because of the addition of collections. Could you update to comment to explain why?

Reply:
That's correct. I will update the comment. Actually while I'm at it ... the existing scratch prefix puts some redundant namespacing in there (something like `/pachyderm_pfs/scratch`) that we don't need now that these keys are namespaced because of the collection.